src/nativeTrackerManager.js

```diff
@@ -67,6 +68,24 @@ export function newNativeTrackerManager(win) {
     }
   }
 
+  function fireNativeImpTracker(adId){
+    let parsedUrl = parseUrl(window.pbNativeData.pubUrl);
+    publisherDomain = parsedUrl.protocol + '://' + parsedUrl.host;
+    fireTracker(adId, 'impression');
+  }
+
+  function fireNativeCallback(){
+    let parsedUrl = parseUrl(window.pbNativeData.pubUrl);
+    publisherDomain = parsedUrl.protocol + '://' + parsedUrl.host;
```

Comment:
We suppose set `publisherDomain` value inside `renderNativeAd` once, instead of set it before each tracker call.

Reply:
Corrected the above to one definition in renderNativeAd as opposed to the tracker definitions

Reply:
Good.